### ISSUE TITLE
RPC: Could not connect

### ISSUE THREAD

**Issue by KittyBorgX** [LeonardSSH/coc-discord-rpc#45]

**Comment by LeonardSSH** [comment]
Try to reload the discord (`CTRL + R`).

or

https://github.com/iCrawl/discord-vscode#troubleshooting

**Comment by KittyX29** [comment]
I reloaded, I am on an M1 Mac Mini.

**Comment by KittyX29** [comment]
still doesnt work

**Comment by KittyX29** [comment]
i tried reinstalling everything also

**Comment by LeonardSSH** [comment]
Try now with the new version of the extension.

**Comment by KittyX29** [comment]
i installed it

**Comment by KittyX29** [comment]
there are no errors, but it doesnt connect

here is a pic when i do :CocComand rpc.enable
![image](https://user-images.githubusercontent.com/82328083/121467658-19076a00-c9d7-11eb-97b6-13fb6196e0b9.png)

**Comment by LuisHenriqueFA14** [comment]
> there are no errors, but it doesnt connect
> 
> here is a pic when i do :CocComand rpc.enable
> ![image](https://user-images.githubusercontent.com/82328083/121467658-19076a00-c9d7-11eb-97b6-13fb6196e0b9.png)

I'm having the same problem ;-;

**Comment by CanKolay5677** [comment]
me too

**Comment by milse113** [comment]
me as well

**Comment by JorgeArreolaS** [comment]
same

**Comment by JorgeArreolaS** [comment]
Hi @milse113 @CanKolay5677 @LuisHenriqueFA14 @KittyBorgX
(sorry for the ping, i dont know about github issues notification system, hope you see it).

I found this error is generated from discord-rcp when is not possible to connect discord's websocket (or something like that), so is not part of the plugin, is (in all the times i've seen it) just an error connection to discord. That's why if you tried another nvim plugins for discord rcp neither worked.

To sove this try to open as administrator in Windows as in this comment of  `sametgirginer`:
[Couldn't connect to Discord via RPC: Error: Could not connect  ##85](https://github.com/iCrawl/discord-vscode/issues/85#issuecomment-417895483)

If your'e on linux, install discord from its official site: [discord](https://discordapp.com/).
As `Isakai`'s comment says: [Install both vscode and discord form their official sites to fix it.](https://github.com/iCrawl/discord-vscode/issues/85#issuecomment-562848483)

In my case, I installed it from Ubuntu Software Installer ( dont use it ) and was the reason of my problem.
I uninstalled and installed from deb file source and its working now.

Hope this solves your problem
